www/jobs.md

```diff
@@ -4,20 +4,15 @@ The Hail team's mission is to build tools to enable rapid analysis and explorati
+Questions? Write us at <a href="mailto:hail@broadinstitute.org"><code>hail@broadinstitute.org</code></a>. **To apply, visit the [offical job post](https://broadinstitute.wd1.myworkdayjobs.com/broad_institute/job/Cambridge-MA/Software-Engineer---Hail-Team_3168)**. For more on our mission and approach, read on.
 
-Two of us ([Cotton Seed](https://www.broadinstitute.org/bios/cotton-seed) and [Jon Bloom](https://www.broadinstitute.org/bios/jonathan-bloom)) co-founded the **Hail project** in the [Neale lab](http://www.nealelab.is/) in Fall 2015 to help the genetics community [harness the flood of sequenced genomes](https://www.broadinstitute.org/blog/harnessing-flood-scaling-data-science-big-genomics-era) in order to unravel the genetic architecture of disease. Our open-source framework is already being used to analyze the largest genetic data sets in existence, to power dozens of major academic studies, and to meet the exploding needs of hospitals, diagnostic labs, and industry. We're now building a new **Initiative in Scalable Analytics** in order to grow the Hail project within and beyond genetics and reduce the latency of computational experiments in biomedical research.
-
-Operationally, we're a software team embedded inside the world's leading biomedical and genomics research institute, anchoring the [global heart of biotech](http://www.wbur.org/bostonomix/2017/06/19/boston-biotech-success) right across from MIT. We implement **distributed algorithms** on top of our custom-built language, compiler, and run-time system to support querying, aggregation, and linear algebra on hundreds of thousands of human genomes. We thrive on **diverse challenges**: language and compiler design, low-level performance optimization, architecture of distributed systems, scaling of established methods and invention of new ones, visualization, interoperability with other powerful tools, and close collaboration with the amazing science and scientists all around us (this is super fun!).
-
-In addition to software development, our team is interested in the application of **machine learning** to problems in biology. We run the [Models, Inference & Algorithms Initiative](http://www.broadinstitute.org/mia) to foster community and pedagogy in greater Boston at the interface of computational biology, mathematical theory, and computer science.
+![](https://storage.googleapis.com/hail-common/hail_team.JPG "Hail team")
```

Comment:
@maccum can I deputize you to wrangle us into a group photo again now that richard has joined?